src/Microsoft.ML.Transforms/Text/StopWordsRemoverTransform.cs

```diff
@@ -238,16 +143,12 @@ private static VersionInfo GetVersionInfo()
                 loaderSignature: LoaderSignature,
                 loaderAssemblyName: typeof(StopWordsRemoverTransform).Assembly.FullName);
         }
+        public IReadOnlyCollection<ColumnInfo> Columns => _columns.AsReadOnly();
```

Comment:
> public I [](start = 7, length = 9)

at least on public members that we add, let's try to have a one line /// <summary>